namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-staging/resources/pingdom.tf

```diff
@@ -0,0 +1,21 @@
+provider "pingdom" {}
+
+terraform {
```

Comment:
You don't need to specify the backend again - it's already set in the main.tf file

Reply:
thank you, I've updated the PR